packages/tests/src/index.ts

```diff
@@ -183,7 +183,7 @@ function execTurborepo(context: GlobalContext) {
   }
 
   const child = exec(npmCli, args, {
-    timeout: 70 * 10 * 1000,
+    timeout: 80 * 10 * 1000,
```

Comment:
@magne4000, is this fine with you ?, It seems that with increasing boilerplate, we also have to increase timeouts.

Reply:
Fine for now yes, I'll optimize this later